pom.xml

```diff
@@ -446,6 +446,33 @@
 					<autoVersionSubmodules>true</autoVersionSubmodules>
 				</configuration>
 			</plugin>
+      
```

Comment:
I'll try to work on adding a profile that will act as a flag for building that stuff or not. More later...